redfish-core/lib/cable.hpp

```diff
@@ -88,6 +88,47 @@ inline void
             }
             resp.jsonValue["CableType"] = *cableTypeDescription;
         }
+        else if (propKey == "CableStatus")
+        {
+            const std::string* cableStatus =
+                std::get_if<std::string>(&propVariant);
+            if (cableStatus == nullptr)
+            {
+                messages::internalError(resp);
+                return;
+            }
+
+            std::string linkStatus = *cableStatus;
+            if (!linkStatus.empty())
```

Comment:
While I code this way in general, but Ed Tanous prefers to return early. His reasoning in the past for me was 'it saves a scope', don't ask me what it means :) 
Can do something like:
if (linkStatus.empty())
{
    continue;
}

Just a nit, but otherwise it looks good!

Reply:
Done.

Reply:
Maybe Ed Tanous is talking about indent level when he says ''it saves a scope'.